string filepath
---
string objectCategory
float64 confident
